fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -143,7 +147,51 @@ private static RetryHandler buildRetryHandler(
   @Override
   public boolean hasPermission(
       Authentication authentication, Object resource, Object authorization) {
-    return false;
+    if (!fiatStatus.isEnabled()) {
+      return true;
+    }
+
+    Resource r;
```

Comment:
This is tricky, but I think we need to think through it a bit more

This code is actually running in-process in the fiat clients, and is annotated onto methods there. (For example in Front50 on a `getApplications` endpoint). In that case, the type is going to be that applications API model type, which isn't going to implement Resource.

Not sure I have a great suggestion immediately, thinking about it..